README.md

```diff
@@ -9,6 +9,7 @@ OSX:
 
 Ubuntu:
  - `sudo apt-get install nodejs`
+ - NOTE: if Node is not found after installation, try running `sudo apt-get install nodejs-legacy` as well
```

Comment:
Probably better would be: "if you later get an error like `node`: command not found, you may need: ...". Great catch though, thanks!